### ISSUE TITLE
Domain users can't use sudo on linux mint 19.2 Xfce

### ISSUE THREAD

**Issue by DenoBeno** [BeyondTrust/pbis-open#232]
**Version**: 8.5.4.334
**OS/Distro**: linux mint 19.2 Xfce
**Issue/Impact**: domain users can't use sudo

I recently installed samba domain controller and checked that it works with a windows machine. Then I installed a pristine linux mint 19.2 Xfce edition, updated the packages, added the pbis-open apt repository, installed pbis and configured it as per instructions in https://linoxide.com/ubuntu-how-to/configure-pbis-join-ubuntu-windows-ad/ 

I am able to log in to the machine with domain user account, but these users can't use sudo.

I double-checked and the domain administrator is in HOME\domain^admins group (local "groups" command correctly returns these groups) and used visudo to add the following line:

%domain^admins ALL=(ALL) ALL

This didn't impress the sudo at all. I also tried with %domain\ admins and with %domain^users - nothing works. Rebooting didn't help.

Any ideas?

**Comment by hydrian** [comment]
I use Linux Mint 19.2 with no sudo problems. 

Use the  "getent group" command to get the exact formatting of your group's identifier. Sometimes domain names are escaped differently than expected.

**Comment by DenoBeno** [comment]
Thanks. I tried that, but it didn't work. No idea why, maybe I misconfigured something but I can't test it anymore because I have reinstalled everything and use realmd and sssd instead of pbis now. 

"realm join" is a really handy command that works pretty well out of the box in linux mint 19.2 and configures almost everything. 

https://www.thegeekdiary.com/how-to-connect-to-an-active-directory-domain-using-realmd-configure-centos-rhel-7-as-active-directory-client/

The remaining issues mostly have to do with user privileges. sudo configuration worked immediately, but fuse has a separate configuration that's somewhat difficult to find.
